src/core/ObservableQuery.ts

```diff
@@ -710,21 +710,19 @@ Did you mean to call refetch(variables) instead of refetch({ variables })?`,
    * @param variables - The new set of variables. If there are missing variables,
    * the previous values of those variables will be used.
    */
-  public async setVariables(
-    variables: TVariables
-  ): Promise<ApolloQueryResult<MaybeMasked<TData>>> {
+  public async setVariables(variables: TVariables) {
```

Comment:
```suggestion
  ): Promise<QueryResult<TData>> {
```

Let's keep these explicit here - while accidental changes will show up in api-explorer diffs, it's just a tad more secure like this, especially in functions with multiple return paths like this.

Reply:
I'm fine with that. Updated in [36dc582](https://github.com/apollographql/apollo-client/pull/12484/commits/36dc5822bc2b04db8d014a7793d5345ec3f83aed)